### PR TITLE
feat(Style): adding content to the style guide (92769)

### PR DESCRIPTION
**Pull request by jeff-colucci**

### CODE REVIEW

**Review by zuluecho9 — APPROVED** [review]

File: src/content/docs/style-guide/structure/pop-overs.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Pop-overs
+tags:
+  - Basic style guide
+  - Style guide quick reference
+---
+import styleguidePopoverexample from 'images/style-guide_screenshot-crop_popover.png'
+
+Pop-overs are a unique component we've created to provide a lot of direction in a small space for our readers. Initially created to help with VSUs, you can use these components to help get readers into the product, go to a document that gives them more information on a subject, or even sign up for New Relic, all in one place!
```
Comment (zuluecho9) [inline src/content/docs/style-guide/structure/pop-overs.mdx:9]: we may not know what VSUs are in future (minor issue)

File: src/content/docs/style-guide/structure/tiles.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+title: Tiles
+tags:
+  - Basic style guide
+  - Style guide quick reference
+redirects:
+---
+
+import styleTileTypes from 'images/style-guide_screenshot-full_tile-examples.webp'
+
+DocTiles, or just Tiles for short, are a unique type of react component designed to provide some visual flair to groups of links. Laid out in a grid, tiles can present a lot of links on a condensed space without overwhelming the reader. These are most useful for docs which direct to numerous places as part of their user journey, such as intro docs.
```
Comment (zuluecho9) [inline src/content/docs/style-guide/structure/tiles.mdx:11]: in keeping with our sentence case style guide, i wonder if we want to capitalize these things or just say 'tabs' or 'tiles' or whatever. in context probably fine to use sentence case

**Review by zuluecho9 — DISMISSED** [review]

File: src/content/docs/style-guide/structure/tiles.mdx (hunk 2 above)
Comment (jeff-colucci) [reply src/content/docs/style-guide/structure/tiles.mdx:11]: Fixed!

**Comment by jeff-colucci** [comment]
Build was successful on manual rebuild, didn't report to GH.

**Pull request merged**